src/components/Display.jsx

```diff
@@ -18,14 +21,20 @@ function Display() {
         <h1>Loading ...</h1>
       ) : (
         <>
-          <h1>Display yatchs!</h1>
+          <h1>Display yachts!</h1>
```

Comment:
**[OPTIONAL]**
The word `yatch` is incorrect. We all fixed that to `yacht`. You can do it too.